src/main/java/com/volasoftware/tinder/repository/UserRepository.java

```diff
@@ -43,4 +43,7 @@ public interface UserRepository extends JpaRepository<User, java.lang.Long> {
     List<FriendDetails> findUserFriendsSortedByLocation(@Param("id") Long id,
                                                         @Param("lat") Double lat,
                                                         @Param("lon") Double lon);
+
+    @Query("SELECT u FROM User u WHERE u.id = :friendId")
+    User findFriendById(@Param("friendId") Long friendId);
```

Comment:
Isn't the same as: 
```suggestion
    User findById(Long friendId);
```
Also, should be Optional, so the method should return Optional<User>